app/components/mandatenbeheer/bestuursorgaan-mandaten.hbs

```diff
@@ -0,0 +1,100 @@
+<div class="au-o-box no-pagination" {{did-insert this.initialize}}>
+  <AuDataTable
+    @content={{this.orderedMandaten}}
+    @isLoading={{this.loading}}
+    @noDataMessage="Geen mandaten gevonden"
+    @page={{0}}
+    @size={{100}}
+    as |t|
+  >
+    <t.content as |c|>
+      <c.header>
+        <th>Naam</th>
+        <th>Aantal Houders</th>
+        <th></th>
+      </c.header>
+      <c.body data-test-loket="mandaten-body" as |mandaat|>
+        <td class="au-u-visible-small-up">
+          {{mandaat.bestuursfunctie.label}}
+        </td>
+
+        <td class="au-u-visible-small-up">
+          {{mandaat.aantalHouders}}
+        </td>
+        <td>
+          <LinkTo
+            @route={{"mandaat.edit"}}
+            @model={{mandaat.id}}
+            class="au-c-link"
+          > Bewerk </LinkTo>
+          <AuButton
+            @skin="link"
+            @icon="bin"
+            @alert="true"
+            @loading={{(and
+              (eq this.removingMandaatId mandaat.id)
+              this.removeMandaatTask.isRunning
+            )}}
+            {{on "click" (fn this.removeMandaat mandaat)}}
+          >
+            Verwijder
+          </AuButton>
+        </td>
+      </c.body>
+    </t.content>
+  </AuDataTable>
+  {{#unless this.bestuursorgaanIT.isDecretaal}}
```

Comment:
This seems to render nonetheless, await missing?

Reply:
this is a classic case of me forgetting the await helper! well spotted!